challenges/objects-arrays.js

```diff
@@ -55,11 +78,23 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
-console.log(contactInfo);
+for (i = 0; i < graduates.length - 1; i++) {
```

Comment:
The `graduates.length - 1` is not doing what you think it is. What is the difference between using `graduates.length` and `graduates.length - 1` in the for loop?